### PR TITLE
Mac installer must add LICENSE to `setup-files`

### LINKED ISSUE #202: Mac installer must add LICENSE to `setup-files`
**Branch:** 0.15.x

**Expected behaviour:** `/installers/osx/setup-files/` folder must have an LICENSE file, so it will be included on the .dmg package.

**Current behaviour:** LICENSE file is not included in setup folder.

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #202

### CODE REVIEW

**Comment by cpauya** [comment]
@amodia Did you try to build a .dmg and checked if this file was included?

**Comment by amodia** [comment]
@cpauya I added the setup-files folder on dmg with a license file and readme.md.

screen shot 2015-10-02 at 6 47 10 pm

screen shot 2015-10-02 at 6 47 20 pm

**Comment by cpauya** [comment]
@amodia The `LICENCE` and `README.md` files must not be inside a folder.  User must be able to read those files without going inside a folder during install.

**Comment by amodia** [comment]
@cpauya I added the `README.md` and `LICENCE` on `more files` directory. 

screen shot 2015-10-06 at 4 55 03 pm

screen shot 2015-10-06 at 4 55 12 pm

screen shot 2015-10-06 at 5 06 01 pm

File: osx/setup.sh [hunk 1]
```diff
@@ -363,8 +364,16 @@ fi
 
 # Remove the .dmg if it exists.
 test -e "$DMG_PATH" && rm "$DMG_PATH"
+
+mkdir "$RELEASE_PATH/More files"
```
Comment (cpauya) [inline osx/setup.sh]: @amodia Check if the "More files" directory already exists before creating, or else this will throw an exception.

**Comment by cpauya** [comment]
Just one minor change @amodia and this should be good to go.

**Comment by amodia** [comment]
@cpauya done creating an if statement to check more files directory exists.

File: osx/setup.sh [hunk 2]
```diff
@@ -363,8 +364,19 @@ fi
 
 # Remove the .dmg if it exists.
 test -e "$DMG_PATH" && rm "$DMG_PATH"
+
+if [ -d "$RELEASE_PATH/More files" ]; then
```
Comment (cpauya) [inline osx/setup.sh]: @amodia Change this into a "constant" -> `"$RELEASE_PATH/More files"` so it's easy to maintain.

File: osx/setup-files/LICENSE [hunk 3]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Foundation for Learning Equality
```
Comment (cpauya) [inline osx/setup-files/LICENSE]: I think we should use 2015, is that correct @aronasorman?

**Comment by cpauya** [comment]
Looks good, merging.

**Pull request merged**